plugins/sigma.renderers.linkurious/canvas/sigma.canvas.nodes.def.js

```diff
@@ -194,6 +206,10 @@
       context.arc(x, y, size, 0, Math.PI * 2, true);
       context.closePath();
       context.fill();
+
+      if (borderSize > 0) {
```

Comment:
yes but that border only operates when the node is *active*.  I don't understand how this *active* state is enabled, or why I care to enable it, but in my graph that code is skipped and the `else` where I inserted my code is executed